### PR TITLE
Feature/hma 4176

### PR DESCRIPTION
**Pull request by chrisob55**
# 📝 Description

  
- [x] Updated CHANGELOG
- [ ] Updated README

# 🛠 Testing Notes

# 📸 Screenshots

### CODE REVIEW

File: components/src/main/res/values/colors.xml [hunk 1]
```diff
@@ -32,7 +32,7 @@
     <color name="hmrc_grey_2">#FFB1B4B6</color>
     <color name="hmrc_grey_3">#FFF3F2F1</color>
     <color name="hmrc_pink">#FFCA2B75</color>
-    <color name="hmrc_yellow">#FFFFDD00</color>
+    <color name="hmrc_yellow">#FFBF47</color>
```
Comment (ngoulongkam) [inline components/src/main/res/values/colors.xml:35]: think you missed out two additional "FF" at the beginning, not sure what/why we need it, but all other ones have it.

File: sample/src/main/res/layout/item_color.xml [hunk 2]
```diff
@@ -51,7 +51,7 @@
             android:layout_height="match_parent"
             android:layout_marginEnd="@dimen/hmrc_spacing_16"
             android:gravity="start|center_vertical"
-            tools:text="Yellow (#FFFFDD00)" />
+            tools:text="Yellow (#FFBF47)" />
```
Comment (ngoulongkam) [inline sample/src/main/res/layout/item_color.xml:54]: same here, assume you need the "FF" at the front.

**Review by ngoulongkam — CHANGES_REQUESTED** [review]

File: components/src/main/res/values/colors.xml (hunk 1 above)
Comment (chrisob55) [reply components/src/main/res/values/colors.xml]: Addressed in 37cfbf34f3f4f1def3882f38baa1ef74fbf5a613

File: sample/src/main/res/layout/item_color.xml (hunk 2 above)
Comment (chrisob55) [reply sample/src/main/res/layout/item_color.xml]: Addressed in 37cfbf34f3f4f1def3882f38baa1ef74fbf5a613

**Review by ngoulongkam — APPROVED** [review]

File: fastlane/README.md [hunk 3]
```diff
@@ -12,7 +12,7 @@ Install _fastlane_ using
 ```
 [sudo] gem install fastlane -NV
 ```
-or alternatively using `brew cask install fastlane`
```
Comment (JamieCruwys) [inline fastlane/README.md:15]: Yes! I kept getting this locally and I'm happy to see this is finally fixed!

**Pull request merged**